### PR TITLE
Collection of small, mostly documentation, fixes

### LINKED ISSUE #4274: I/O Ref description of Pump flow rate schedule should explain that it sets max flow not actual flow
User complains that his pump flow rate schedule to vary flow on a variable speed condenser pump is not working.  Comments in ticket say that condenser pump needs to be on supply side of Condenser Loop but user has them on the demand side "after" the chiller condenser.  Ticket says that there should be an error message issued but when you run the file with x8.1 the file runs successfully and no such error is issued.
Helpdesk Ticket 8828
User file
AM3_Testing2.idf

### LINKED ISSUE #4392: Question about Material:RoofVegetation Object and HeatBalanceAlgorithm choice
Helpdesk Ticket 9621

MJW Comment to user question
Should probably add this equation to the Engineering Ref:

Here is the equation in the source code for sigmaf as a function of LAI:
 sigmaf = 0.9d0 - 0.7d0*exp(-0.75d0*LAI) ! Fractional veg cover based on (2) from FASST TR-04-25
 ! Formula for grasses modified to incorporate limits from
 ! Table 1 for sigmaf_max and min (0.20 to 0.9)

User Questions
> To whom it may concern,
 > Hi,
 > This is Hoda Ganji, Researcher and graduate student
 > from master of "Energy and Architecture", university of Tehran,
 > Tehran, Iran. Currently, I am working on a paper for "Building and
 > Environment" Journal and I have a few questions about the object
 > "Material:RoofVegetation":
 > 1-     According
 > to the InputOutputReference, at "HeatBalanceAlgorithm" part,
 > it seems reasonable to use "CombinedHeatAndMoistureFiniteElement" for
 > Material:RoofVegetation in order to calculate the effect of moisture
 > in the
 > air gap layer between the exterior envelope and the green layer.
 > However,
 > according to the EngineeringRefrence, the calculations for
 > Material:RoofVegetation is based upon CTF (ConductionTransferFunction)
 > not
 > CHAMFE. Now, my question is would there be any problem if I select
 > CombinedHeatAndMoistureFiniteElement?

 The Material:RoofVegetation model only makes a sensible heat connection with the rest of the model, so connecting it to HAMT will not accomplish anything. The soil moisture and evaporation are all part of the green roof energy balance, but it does not transmit moisture to the building surface below.

 > I mean, do these calculations work only if CTF is selected or can they
 > still be
 > applied in CHAMFE situation?
 > 2-     My
 > other question is that In InputOutputRefrence, it has been mentioned
 > that for Field: Leaf Area Index: 
 > At the present time the fraction vegetation cover is
 > calculated directly from LAI (Leaf Area Index) using an empirical
 > relation. The
 > user may find it necessary to increase the specified value of LAI in
 > order to represent
 > high fractional coverage of the surface by vegetation.
 > Thus, although "fraction vegetation cover" is
 > quite a consequential variable, there is no need to calculate it
 > separately,
 > right? The text says that it is possible to increase the specified LAI
 > value
 > in order to represent high fractional coverage. Now, how much should I
 > add
 > to LAI value if the fraction coverage is relatively high?

### LINKED ISSUE #4418: Green roof simulation stability check suggests using 120 timesteps per hour
Helpdesk Ticket 9038

Ecoroof stability check needs better documentation - looks to me like there's a certain minimum depth at which 60 timesteps become insufficient and you should switch to the simple method? Also, the error check needs to check and not tell you to use something greater than 60.  This could just be some updates to the error message.

User file Green roof simulation.idf - can run with any epw to see the error.

 Michael J. Witte on 23 January 2014 08:15 PM

Well, that's interesting. Clearly the error needs to check against the maximum possible number of timesteps. Is this important? I'm not really sure. The simulation will run even though this stability test fails. Here is the test:

 !This loop outputs the minimum number of time steps needed to keep the solution stable
 !The equation is minimum timestep in seconds=161240*(2)**(-2.3))*(Total thickness of the soil)**2.07

 This test is only used when the Moisture Diffusion Calculation Method = Advanced. If I am following this correctly, this problem occurs when the Material:RoofVegetation Thickness goes below a certain value. So, for thinner soil layers, you should probably switch to the Simple moisture diffusion calculation method.

 Mike

> I am simulating a green roof and I get this severe error:
 > ............
 > Program Version,EnergyPlus-Windows-64 8.1.0.008, YMD=2014.01.23
 > 11:04,IDD_Version 8.1.0.008
 > 
 > ** Severe ** CalcEcoRoof: Too few time steps per hour for stability.
 > ** ~~~ ** ...Entered Timesteps per hour=[60], Change to some value
 > greater than [120] for assured stability.
 > ..........
 > 
 > The program tells me to increase the time steps but the maximum value
 > I can introduce is 60. Could you please help me?
 > 
 > Thank you very much.

### LINKED ISSUE #4542: Auxiliary Programs document section on Transition has old version references
In the Auxiliary Programs document, the Transition section has references to V7.0  We may want to revise this section to be more generic.  e.g. "TransitionV6-0-0-to-V7-0-0.exe is the current transition program that is distributed with the V7.0 release."

Also, there is a reference error in this doc: "The IDF Version Updater lives in the folder with the multiple transition programs -- see Error! Reference source not found"

### LINKED ISSUE #4545: I/O Ref - DesignSpecification:ZoneHVAC:Sizing is misplaced
Helpdesk Ticket 9927

In EnergyPlus 8.2, a new object "DesignSpecification:ZoneHVAC:Sizing" was added. In IDF Editor, the object shows up in the group "HVAC Design Objects", however, in I/O Reference document, it is placed in the group "Zone Forced Air Units". This creates some confusion for people to find the documentation.

Also noticed there are several occurences of "DesignSpecificationZoneHVACSizing" which should be "DesignSpecification:ZoneHVAC:Sizing".

### LINKED ISSUE #4925: EnergyPlus can't be installed on Ubuntu 14.04 if /usr/local/share/man/man1/ does not exist.
On Linux 64 (Ubuntu), EnergyPlus-8.3.0 relies on the directory /usr/local/share/man/man1 to be installed. If the directory doesn't exist then the installation will fail. If the directory is manually created, then the installation will successfully complete but EnergyPlus will still fail to install because of wrong symbolic links. See the email thread below to reproduce the problem.

Thierry

---

Does any of you succeed in installing the latest release of  EnergyPlus on Ubuntu 14.04?

When I tried to install it using the binaries on 

https://github.com/NREL/EnergyPlus/releases/download/v8.3.0/EnergyPlus-8.3.0-6d97d074ea-Linux-x86_64.sh

I got 

---

Extracting, please wait...
Unpacking to directory /usr/local was successful.
## mv: cannot move '/usr/local/EnergyPlus-8-3-0/energyplus.1' to '/usr/local/share/man/man1/': Not a directory

then I manually created the directory "man1" (which I believe I shouldn't) and tried again. 

This time I got 

---

Extracting, please wait...
Unpacking to directory /usr/local was successful.
## Symbolic links were successful.

But when I tried to start EnergyPlus I got 

---

thierry@srgbcvtb2:~/Downloads$ EnergyPlus
ERROR: Could not find input data dictionary: /home/thierry/Downloads/Energy+.idd.
## Type 'energyplus --help' for usage.

EnergyPlus was basically looking for the IDD at the wrong place.

Are those issues known?

Thanks!

### PR DESCRIPTION
**Pull request by Myoldmopar**
So far:
- Update output variable parser to report bad lines;
- Update source in a few spots so SetupOutputVariable occurs on a single line for easier parsing;
- Clean up gobs and gobs of bad characters in 10-Sizing that were keeping the pdfs from building (pandoc failure);
- Fixes #4925 by creating the man page folder before trying to copy a file in there
- Fixes #4542 by generalizing the transition program documentation and cleaning up a bad reference
- Fixes #4545 by moving the ZoneHVAC sizing spec object and cleaning up some object names (only in documentation)
- Fixes #4274 by clarifying the pump flow rate schedule field
- Fixes #4418 by adding text in the warning message if the ecoroof required timestep is higher than currently used/available
- Fixes #4392 by adding equation for sigma_f = f(LAI) into engineering reference

Partially addresses #5190  (Eventually will cover the code-related changes required to wrap up #5190 and perform the public release.)

### CODE REVIEW

**Comment by Myoldmopar** [comment]
@mjwitte I've got a list going here of a whole bunch of super tiny changes for this PR.  I'd like to get it merged in quickly and move on.  The rest of the bug fixes will require unit tests, etc., so they will each be in their own.  This was just a collection of incredibly small ones to lump together.

These are so small, I'm comfortable merging it myself if you don't want to spend time reviewing this one.

**Comment by mjwitte** [comment]
@Myoldmopar Check the line endings here.
doc/src/docs/EngineeringReference/10-Sizing.md
289 additions, 289 deletions not shown

**Comment by Myoldmopar** [comment]
Thanks @mjwitte, I'll check. Those should be valid though. Just replacing invalid characters.

**Comment by mjwitte** [comment]
@Myoldmopar I've reviewed everything else.  Assuming 10-Sizing.md is ok and the CI comes back OK, I'll merge this in.

**Comment by Myoldmopar** [comment]
10-Sizing.md looks perfect on my machine.  No line ending problem.  Thanks.

File: src/EnergyPlus/EcoRoofManager.cc [hunk 1]
```diff
@@ -732,12 +732,19 @@ namespace EcoRoofManager {
...
 				Real64 const depth_limit( depth_fac * std::pow( TopDepth + RootDepth, 2.07 ) );
 				for ( index1 = 1; index1 <= 20; ++index1 ) {
-					if ( double( MinutesPerTimeStep / index1 ) <= depth_limit ) break; //Autodesk RootDepth was used uninitialized here
+					if ( double( MinutesPerTimeStep / index1 ) <= depth_limit ) break;
 				}
 				if ( index1 > 1 ) {
-					ShowSevereError( "CalcEcoRoof: Too few time steps per hour for stability." );
-					ShowContinueError( "...Entered Timesteps per hour=[" + RoundSigDigits( NumOfTimeStepInHour ) + "], Change to some value greater than [" + RoundSigDigits( 60 * index1 / MinutesPerTimeStep ) + "] for assured stability." );
-					//      CALL ShowFatalError('Program terminates due to previous condition.')
+					ShowWarningError( "CalcEcoRoof: Too few time steps per hour for stability." );
```
Comment (mjwitte) [inline src/EnergyPlus/EcoRoofManager.cc:738]: Subtle - changing from severe to warning  . . . . I'll let it slide here.

**Comment by mjwitte** [comment]
So, shouldn't we change the eco roof example file to avoid the timestep too low error?

**Comment by Myoldmopar** [comment]
Sure can!  Let me do that now.

**Comment by Myoldmopar** [comment]
OK, verified that the modified eco roof file no longer gives warnings with the timestep cranked up.  This causes a longer runtime, of course.  And will also cause some numeric diffs, of course.  I also added a minor documentation addition to fix another doc-only-issue.  This should be good to go once CI is done with a28fd63.

**Comment by Myoldmopar** [comment]
OK, things are good.  The Ecoroof file has diffs as expected.  This should be merged after a couple more come back.

Thanks for reviewing this @mjwitte.

**Comment by Myoldmopar** [comment]
All green, merge at will, @mjwitte

**Pull request merged**